src/customMetadata/frField.campaignPageName.md

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>Campaign Page Name</label>
+    <protected>false</protected>
```

Comment:
what does <protected> mean?

Reply:
https://trailhead.salesforce.com/en/content/learn/modules/custom_metadata_types/custom_metadata_types_manageability
Looks like this determines whether custom metadata is available outside the managed package or not. In our case it would be correct to leave these as public (<protected>false</protected>)